libpod/runtime.go

```diff
@@ -42,23 +42,32 @@ const (
 	SQLiteStateStore RuntimeStateStore = iota
 	// BoltDBStateStore is a state backed by a BoltDB database
 	BoltDBStateStore RuntimeStateStore = iota
+)
+
+var (
+	// InstallPrefix is the prefix where podman will be installed.
+	// It can be overridden at build time.
+	installPrefix string = "/usr/local"
+	// EtcDir is the sysconfdir where podman should look for system config files.
+	// It can be overridden at build time.
+	etcDir string = "/etc"
```

Comment:
Go Lint does not like the `string` here.
etcDir = "/etc"
is preferred.

Reply:
Fixed